docs/reference/standard-library/std/coins.md

```diff
@@ -0,0 +1,50 @@
+---
+id: coins
+---
+
+# Coins
+
+`Coins` is a set of `Coin`, one per denomination.
+
+```go
+type Coins []Coin
+func (cz Coins) String() string {...}
+func (cz Coins) AmountOf(denom string) int64 {...}
+func (a Coins)  Add(b Coins) Coins {...}
```

Comment:
I think the receiver should have the same value for all methods.  If that's the case then maybe rename `b` to `c`, or better yet, `coins`.

Reply:
This is how they are defined in the VM itself. You have a point - I will fix them in the docs, and we should probably do so as well in the VM itself, for consistency and readability.

Reply:
[7f069c3](https://github.com/gnolang/gno/pull/1468/commits/7f069c3989c1e5bc71e1a9d79cc7fd97d7d1a95e)

Reply:
[3012f21](https://github.com/gnolang/gno/pull/1468/commits/3012f21b1f833de06fe6e247608464477e14d7c7)